src/Mailer/Message/CertificationRequestApprovedMessage.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Mailer\Message;
+
+use App\Entity\CertificationRequest;
+use Ramsey\Uuid\Uuid;
+
+final class CertificationRequestApprovedMessage extends Message
+{
+    public static function create(CertificationRequest $certificationRequest): self
+    {
+        $adherent = $certificationRequest->getAdherent();
+
+        return new self(
+            Uuid::uuid4(),
+            $adherent->getEmailAddress(),
+            $adherent->getFullName(),
+            'Vous êtes certifié',
+            [],
+            static::getRecipientVars($adherent->getFirstName())
```

Comment:
as an alternative without additional method:
 
```suggestion
            ['first_name' => self::escape($adherent->getFirstName())]
```